packages/notifications/src/styled/StyledAlert.ts

```diff
@@ -5,37 +5,157 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { retrieveComponentStyles, getColor } from '@zendeskgarden/react-theming';
 import { VALIDATION_TYPE } from '../utils/types';
+import { StyledTitle, StyledClose } from '../styled';
 
 const COMPONENT_ID = 'notifications.alert';
 
 export interface IStyledAlertProps {
   type?: VALIDATION_TYPE;
 }
 
+const closeStyles = (props: IStyledAlertProps & ThemeProps<DefaultTheme>) => {
+  const { type, theme } = props;
+  const { colors, palette } = theme;
+  const { successHue, dangerHue, warningHue, neutralHue } = colors;
+
+  let color;
+  let pseudoColor;
+  let bgColor;
+
+  switch (type) {
+    case 'success':
+      color = palette[successHue][500];
+      pseudoColor = palette[successHue][800];
+      bgColor = getColor(successHue, 600, theme, 0.15);
+      break;
+    case 'error':
+      color = palette[dangerHue][500];
+      pseudoColor = palette[dangerHue][800];
+      bgColor = getColor(dangerHue, 600, theme, 0.15);
+      break;
+    case 'warning':
+      color = palette[warningHue][700];
+      pseudoColor = palette[warningHue][800];
+      bgColor = getColor(warningHue, 700, theme, 0.15);
+      break;
+    case 'info':
+      color = palette[neutralHue][600];
+      pseudoColor = palette[neutralHue][800];
+      bgColor = getColor(neutralHue, 600, theme, 0.15);
```

Comment:
> Furthermore, I ran the notion of a flexible color hue application by design and got resoundingly positive feedback.

We spoke about the notion of a flexible color hue application a few days ago. I walked away not fully understanding, and that was my fault — I should have asked for an example. What’s an example of flexible color hue application? And what is an example of in-flexible color hue application? Could we highlight the in-flexible color hue application in this PR?

Reply:
I'm not familiar with any non-validation background colors used in product. Why would we need to start supporting this now?

Reply:
Marking as resolved because this is closely related to the [thread ](https://github.com/zendeskgarden/react-components/pull/563#discussion_r359993043)about a flexible `StyledDiv` component.